dali/base/dasds.cpp

```diff
@@ -4091,12 +4088,12 @@ bool translateOldFormat(CServerRemoteTree *parentServerTree, IPropertyTree *pare
 
 void CSDSTransactionServer::processMessage(CMessageBuffer &mb)
 {
-    // 1) Decrement sds_pending_requests here.
+    // 1) Increment sds_requests.started here.
     // NB: request has started, no longer pending
-    // See CSDSTransactionServer::run() for where needs to be incremented. 
+    // See CSDSTransactionServer::run() for where needs to be incremented.
 
-    // 2) Ensure sds_active_requests is decremented, when processMessage() is finished
-    // e.g. use a scoped object here, and decrement in dtor.
+    pSdsRequestsStarted->inc(1);  // this is where the active count was incremented before
+    COnScopeExit incCompletedOnExit([&](){ pSdsRequestsCompleted->inc(); });
```

Comment:
inc(1) vs inc() inconsistency.

Unless @ghalliday disagrees, let's take out the default param value altogether and force it to [consistently] require to be inc(1)

Reply:
Since that is change to the base framework, I would like to address it in a separate Jira after this is merged. @ghalliday, comments?

Reply:
I don't mind if this pr, or the next.  The next should also revisit all the existing metric names and make sure they match the agreed naming scheme.  (I think they may already, but worth double checking.)

Reply:
Opened https://track.hpccsystems.com/browse/HPCC-26501 to address this.

Reply:
>Since that is change to the base framework, I would like to address it in a separate Jira after this is merged.

without chanigng the methods, can we at least make the call consistent? i.e. make both the above calls inc(1).